src/button/interfaces.ts

```diff
@@ -81,6 +81,12 @@ export interface ButtonProps extends BaseComponentProps {
    */
   target?: string;
 
+  /**
```

Comment:
The description to the property is pending content review, will update once it gets finalised.

Reply:
Reviewed and approved by content team.